$NetBSD: patch-plugins_CMakeLists.txt,v 1.1 2021/02/20 20:53:01 ryoon Exp $

* plugins directory has no children at this moment.
  Fix cmake error.

--- plugins/CMakeLists.txt.orig	2020-09-21 00:43:22.000000000 +0000
+++ plugins/CMakeLists.txt
@@ -18,7 +18,7 @@ if(DEFINED ENV{RIME_PLUGINS})
 else()
   file(GLOB plugin_files "*")
   foreach(file ${plugin_files})
-    if(IS_DIRECTORY ${file})
+    if(IS_DIRECTORY ${file} AND EXISTS "${file}/CMakeLists.txt")
       message(STATUS "Found plugin: ${file}")
       set(plugins ${plugins} ${file})
     endif()
